charts/pulsar-operator/README.md

```diff
@@ -1,76 +1,139 @@
-# Pulsar Operator Helm Chart
+# StreamNative Pulsar Operators
 
-The `charts/pulsar-operator` is the the officially supported Helm Chart for installing Pulsar operators. The `charts/pulsar-operator` provides Customer Resource Definitions (CRDs) and Controllers to manage and run Pulsar clusters in a more resilient way.
+The Pulsar Operators bring the specific controllers for Kubernetes by providing specific Custom Resource Definition (CRD) which is developed and maintained by StreamNative Inc.
+Installing the Pulsar Operators means you agreed to and are in compliance with [StreamNative Community License](https://streamnative.io/community-licence).
 
 ## Requirements
 
-To use the `pulsar-operator` chart to deploy BookKeeper Controller, ZooKeeper Controller, and Pulsar Manager Controller, the followings are required.
+To use the `pulsar-operator` chart to deploy BookKeeper Controller, ZooKeeper Controller, and Pulsar Controller, the followings are required.
 
 - Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) 1.16 or higher, compatible with your cluster (+/- 1 minor release from your cluster).
 - Install [`helm`](https://helm.sh/docs/intro/install/) v3 (3.0.2 or higher).
-- Prepare a Kubernetes cluster, version 1.16 or higher.
+- Prepare a Kubernetes cluster, version 1.16 to 1.25. 
 
 ## Install `pulsar-operator` chart
 
 1. Create a Kubernetes namespace.
 
     ```
-    kubectl create namespace <k8s-namespace>
+    kubectl create namespace pulsar
     ```
 
-2. Install the `pulsar-operator` chart.
+2. Add the `streamnative` repo.
+  
+    ```
+    helm repo add streamnative https://charts.streamnative.io
+    helm repo update
+    ```
+
+3. Install the `pulsar-operator` chart.
+
+    ```
+    helm install pulsar-operators streamnative/pulsar-operator --namespace pulsar
+    ```
+
+4. Verify that the `pulsar-operator` chart is installed successfully.
+
+    ```
+    kubectl get po -n pulsar
```

Comment:
kubectl get pod -n <k8s-namespace>
And we should keep consistent in the doc: using `-n <k8s-namespace> or --namespace <k8s-namespace>`

Reply:
I changed the `-n <k8s-namespace>` to hard code as `pulsar` because of the [quick-start-examples](https://github.com/streamnative/charts/blob/master/examples/pulsar-operators/quick-start.yaml#L25)